app/src/main/java/com/example/triptracker/model/repository/UserProfileRepository.kt

```diff
@@ -138,6 +146,26 @@ open class UserProfileRepository {
     return favoritesPaths
   }
 
+  private fun createProfilePrivacy(id: String): Int {
+    val profilePrivacy = 0
```

Comment:
Maybe later we can change these values to an enum type (for cleaner code but not urgent)